README.md

```diff
@@ -1,8 +1,55 @@
 # Sinon Test
-> Test Framework helpers for SinonJS
+> Automatic sandbox setup and teardown for SinonJS
 
-## Help wanted
-We could need a hand maintaining this project to enable us to focus on the core Sinon project. [Want to help us out](https://github.com/sinonjs/sinon-test/issues/12)? 
+## Why?
+Instead of writing tedious setup and teardown code for each
+individual test case you can let Sinon do all the cleanup for you.
+
+So instead of doing this (using [Mocha](https://mochajs.org/) syntax):
+
+```javascript
+var spy1;
+var spy2;
+
+afterEach(()={
+    spy1.restore();
+    spy2.restore();
+});
+
+it('should do something', ()=>{
+    spy1 = sinon.spy(myFunc);
+    spy2 = sinon.spy(myOtherFunc);
+    myFunc(1);
+    myFunc(2);
+    assert(spy1.calledWith(1));
+    assert(spy1.calledWith(2));
+});
+```
+
+You could write just this
+
+```javascript
+it('should do something', sinon.test(function(){
```

Comment:
It should work. As far as I know lambdas have a length property and an apply method (even though the context is ignored).